kDrive/AppDelegate.swift

```diff
@@ -221,6 +221,19 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, AccountManagerDeleg
                 StorePresenter.showStore(from: viewController, driveFileManager: driveFileManager)
```

Comment:
All the code related to the parsing of `deeplinks` (URL components code here) should be split in a dedicated type that lives in drive-core. That way the app delegate stays clean.

Then we should think about abstracting it and sharing with drive, so we have the benefits of this for all our apps.